### PR TITLE
release 1.13.0

### PR DESCRIPTION
**Pull request by philbrookes**

### CODE REVIEW

File: deploy/olm-catalog/integreatly-operator/integreatly-operator-1.13.0/integreatly-operator.v1.13.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+            "namespacePrefix": "integreatly-",
+            "selfSignedCerts": "SELF_SIGNED_CERTS",
+            "type": "INSTALLATION_TYPE"
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: Integreating greatly
+    certified: "false"
+    containerImage: quay.io/integreatly/integreatly-operator:v1.12.0
```
Comment (pawelpaszki) [inline deploy/olm-catalog/integreatly-operator/integreatly-operator-1.13.0/integreatly-operator.v1.13.0.clusterserviceversion.yaml]: should the version here be v1.13.0?

File: deploy/olm-catalog/integreatly-operator/integreatly-operator-1.13.0/integreatly-operator.v1.13.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+  keywords:
+  - integreatly
+  labels:
+    alm-owner-integreatly: integreatly-operator
+    operated-by: integreatly-operator
+  maintainers:
+  - email: integreatly-support@redhat.com
+    name: Integr8ly
+  maturity: alpha
+  minKubeVersion: 1.11.0
```
Comment (davidffrench) [inline deploy/olm-catalog/integreatly-operator/integreatly-operator-1.13.0/integreatly-operator.v1.13.0.clusterserviceversion.yaml]: @philbrookes Sorry, missed this on the first pass. This is no longer the case with the new CRD. min version is 1.12.0

File: deploy/olm-catalog/integreatly-operator/integreatly-operator-1.13.0/integreatly-operator.v1.13.0.clusterserviceversion.yaml (hunk 1 above)
Comment (philbrookes) [reply deploy/olm-catalog/integreatly-operator/integreatly-operator-1.13.0/integreatly-operator.v1.13.0.clusterserviceversion.yaml]: yeah it should. It did update where it mattered, not sure why it didn't update here.

Comment (philbrookes) [reply deploy/olm-catalog/integreatly-operator/integreatly-operator-1.13.0/integreatly-operator.v1.13.0.clusterserviceversion.yaml]: pushed a fix now, anyway

File: deploy/olm-catalog/integreatly-operator/integreatly-operator-1.13.0/integreatly-operator.v1.13.0.clusterserviceversion.yaml (hunk 2 above)
Comment (philbrookes) [reply deploy/olm-catalog/integreatly-operator/integreatly-operator-1.13.0/integreatly-operator.v1.13.0.clusterserviceversion.yaml]: nice catch, fixed now.

**Comment by davidffrench** [comment]
I am curious for the future @philbrookes . What steps did you do to generate the CSVs? (I haven't done it before). and what changes in this PR were manual. I suppose a better way of asking, what are all the steps you took to create a release for this operator and are they documented anywhere?

**Comment by philbrookes** [comment]
@davidffrench I modfied the tags in the makefile manually, then ran `make gen/csv` the two comments on this PR were also addressed manually.

**Comment by davidffrench** [comment]
> @davidffrench I modfied the tags in the makefile manually, then ran `make gen/csv` the two comments on this PR were also addressed manually.

Thank you @philbrookes

**Comment by KevFan** [comment]
@philbrookes Looks like there is 2 package.yaml file : `integreatly.package.yaml` and `integreatly-operator.package.yaml` which looks to be a new one generated from the new operator-sdk version. Might want to rename the old one or which switch over to the new one and update the `make gen/csv` command to target the new file name - https://github.com/integr8ly/integreatly-operator/blob/b43c00b120b53024887dc5982321ff6ee5ff7ae0/Makefile#L213

Other question is do we dont bump the version in `version.go` anymore? :thinking: https://github.com/integr8ly/integreatly-operator/blob/b43c00b120b53024887dc5982321ff6ee5ff7ae0/version/version.go#L4

**Comment by philbrookes** [comment]
> @philbrookes Looks like there is 2 package.yaml file : `integreatly.package.yaml` and `integreatly-operator.package.yaml` which looks to be a new one generated from the new operator-sdk version. Might want to rename the old one or which switch over to the new one and update the `make gen/csv` command to target the new file name -
> 
> https://github.com/integr8ly/integreatly-operator/blob/b43c00b120b53024887dc5982321ff6ee5ff7ae0/Makefile#L213
> 
> Other question is do we dont bump the version in `version.go` anymore? thinking
> 
> https://github.com/integr8ly/integreatly-operator/blob/b43c00b120b53024887dc5982321ff6ee5ff7ae0/version/version.go#L4

Excellent catch @KevFan I've addressed both of those issues now.

**Pull request merged**